resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+restartfile=/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/restartcounter.txt
+cur=$(head -n 1 $restartfile)
+initial=$cur
+timeout="300"
+
+if [ -e $restartfile ];  then
+    ${JAVA_BIN} ${JAVA_ARGS} -Djava.security.egd=/dev/urandom -cp ${INSTALL_DIR}/<%= EZBake::Config[:uberjar_name] %> clojure.main -m <%= EZBake::Config[:main_namespace] %> --config "${CONFIG}" -b "${BOOTSTRAP_CONFIG}" &
```

Comment:
This looks right. I didn't test this outside of running from init scripts with ExecStart/daemon though, so it may need some additional work if it's also intended for stand-alone work (not sure if it will correctly not kill the java process).